ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -46,7 +46,9 @@ port_profiles:
   < port_profile_1 >:
     parent_profile: < port_profile_name >
     speed: < interface_speed | forced interface_speed | auto interface_speed >
+    enabled: < true | false >
```

Comment:
From the heading above:
```yaml
# Each profile can support all or some of the following keys according to your own needs.
# Keys are the same used under endpoints adapters. Keys defined under endpoints adapters take precedence.
```
I don't think we need to maintain the full list of keys in two places. Maybe we could even cut down the port_profile to  `< endpoint_adapter_config >` or something.